### PR TITLE
Allow UE3 games to startup

### PR DESCRIPTION
**Pull request by itmo153277**
Basically this allows SARPBC to startup (see discussion [here](http://www.emunewz.net/forum/showthread.php?tid=176091)) Also it seems to fix this [issue](https://github.com/RPCS3/rpcs3/issues/2220)

### CODE REVIEW

File: rpcs3/Emu/Cell/Modules/cellGame.cpp [hunk 1]
```diff
@@ -195,9 +195,12 @@ s32 cellHddGameExitBroken()
 }
 
 
-s32 cellGameDataGetSizeKB()
+s32 cellGameDataGetSizeKB(vm::ptr<u32> size)
 {
-	fmt::throw_exception("Unimplemented" HERE);
+	if (size) {
+		*size = 0;
```
Comment (paulsapps) [inline rpcs3/Emu/Cell/Modules/cellGame.cpp]: ?? this still isn't implemented! Finding that this is the cause of a failure in the future will require some voodoo skills, surely you should at least log out that this is a hack/stub?

Comment (itmo153277) [inline rpcs3/Emu/Cell/Modules/cellGame.cpp]: Totally agree. Didn't find any info on this function in docs, still kind of mystery for me what it should be doing.

**Comment by AniLeo** [comment]
Now gives me CELL_VIDEO_OUT_ERROR_ILLEGAL_CONFIGURATION instead of CELL_VIDEO_OUT_ERROR_ILLEGAL_PARAMETER on SARPBC

**Comment by itmo153277** [comment]
Can you send log file?

**Comment by AniLeo** [comment]
@itmo153277 
[RPCS3.zip](https://github.com/RPCS3/rpcs3/files/633843/RPCS3.zip)

**Comment by itmo153277** [comment]
No, that's not the case. The reason why you get ILLEGAL_CONFIGURATION is because I limited available configurations to the one user selected in options. If the game doesn't like since the only available option it just doesn't know what to choose. For example, in your case it doesn't like 1080p so it tries 720p even though it is not an available option. If aspect ratio and resolution don't mach it will send VIDEO_RESOLUTION_UNDEFINED instead.
Anyway, game does start up ignoring ILLEGAL_CONFIGURATION error it seems. It doesn't show anything because of cellGcmSetFlipImmediate which I am working on now

**Comment by Nekotekina** [comment]
If this PR doesn't break anything, it must be OK to merge.

**Comment by MarioSonic2987** [comment]
UE3 means Unreal Engine 3?

**Pull request merged**